docs/installation/linux/ubuntulinux.md

```diff
@@ -62,7 +62,7 @@ packages from the new repository:
 
 4. Add the new `GPG` key.
 
-        $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+        $ sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
 
```

Comment:
The reason for using p80 (and the explicit port 80) is that some people can't access servers not on port 80 for firewalling reasons. It is fine to use `ha` if it works for you but we wanted something more general...

(fairly sure this was my first commit to the project last year!)